plugins/commands/install.go

```diff
@@ -185,19 +184,16 @@ func downloadPlugin(pluginsDir, pluginName, downloadUrl string, httpDetails http
 			}
 		}()
 	}
-	log.Info("Downloading plugin: " + pluginName)
 
-	resp, err := client.DownloadFileWithProgress(downloadDetails, "", httpDetails, false, progressMgr)
+	err = downloadPluginExec(downloadUrl, pluginName, pluginsDir, httpDetails, progressMgr)
 	if err != nil {
 		return
```

Comment:
We better delete the local plugin dir before returning.

Reply:
why delete? we want the resources dir to be in our local plugins dir